### PR TITLE
feat(design-land): added checkbox examples to design-land

### PR DESCRIPTION
**Pull request by Nolan-Arnold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

**Comment by Nolan-Arnold** [comment]
This has been rebased and no longer has the intial `DaffCheckboxComponent` commit.

File: apps/design-land/src/app/checkbox/checkbox.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<basic-checkbox></basic-checkbox>
```
Comment (xelaint) [inline apps/design-land/src/app/checkbox/checkbox.component.html:1]: This needs to use `design-land-example-viewer`?

File: apps/design-land/src/app/checkbox/checkbox.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+<br><br><br>
+<checkbox-set></checkbox-set>
```
Comment (xelaint) [inline apps/design-land/src/app/checkbox/checkbox.component.html:3]: This needs to use `design-land-example-viewer`?

File: apps/design-land/src/app/checkbox/checkbox.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+<br><br><br>
+<generated-set></generated-set>
```
Comment (xelaint) [inline apps/design-land/src/app/checkbox/checkbox.component.html:5]: This needs to use `design-land-example-viewer`?

File: apps/design-land/src/app/checkbox/examples/generated_set/generated_set.component.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { FormArray, FormControl } from '@angular/forms';
+import { DaffCheckboxSetComponent } from '@daffodil/design';
+
+@Component({
+    selector: 'generated-set',
```
Comment (xelaint) [inline apps/design-land/src/app/checkbox/examples/generated_set/generated_set.component.ts:6]: Is an example for this needed? Generating a checkbox set from a list with an array isn't a UI thing that needs to be outlined? We already have the example of what a checkbox-set is.

**Pull request merged**